sdk/lib/opentelemetry/sdk/trace/span.rb

```diff
@@ -264,39 +264,46 @@ def trim_span_attributes(attrs)
           nil
         end
 
-        def trim_links(links, max_links_count, max_attributes_per_link) # rubocop:disable Metrics/AbcSize
+        def trim_links(links, max_links_count, max_attributes_per_link) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
Deferring checks to the SDK seems like the right choice, but Rubocop agrees that the resulting code *is* complex.

Reply:
I also agree on both points!